docs/guides/setup-dbt.mdx

```diff
@@ -1,257 +1,103 @@
-2. In the file browser that pops up, click the file named
-   `my_second_dbt_model.sql` under the folders `demo/models/example/`. 
-   
-   1. This will add 2 dbt blocks to your pipeline: 1 for the dbt model named
-   `my_first_dbt_model` and the 2nd for the dbt model named `my_second_dbt_model`. 
-   
-   2. The model named `my_first_dbt_model` was added to
-   the pipeline because `my_second_dbt_model` references it.
+## Prerequisites
 
-![](https://github.com/mage-ai/assets/blob/main/dbt/add-dbt-model.gif?raw=true)
+1. [Docker](https://docs.docker.com/engine/install/)
+2. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
-### 6a. Edit dbt model `my_first_dbt_model`
+## Configuration
 
-1. In the dbt block named `my_first_dbt_model`, next to the label
-   `Target` at the top, choose `dev` in the dropdown list.  You can also
-   check `Manually enter target`, and enter `dev` in the input field.
-2. Paste the following SQL into the dbt model named `my_first_dbt_model`:
+The following command will clone the [demo repo](https://github.com/mage-ai/dbt-quickstart), copy `dev.env` to `.env` and run `docker compose up` to start Mage _and_ a Postgres database.
 
-   ```sql
-   WITH source_data AS (
-       SELECT 1 AS id
-       UNION ALL
-       SELECT 2 AS id
-   )
+<CodeGroup>
+```bash Mac/Linux
+git clone https://github.com/mage-ai/dbt-quickstart mage-dbt-quickstart \
+&& cd mage-dbt-quickstart \
+&& cp dev.env .env && rm dev.env \
+&& docker compose up
+```
 
-   SELECT *
-   FROM source_data
-   ```
+```bash Windows
+git clone https://github.com/mage-ai/dbt-quickstart mage-dbt-quickstart 
+cd mage-dbt-quickstart
+cp dev.env .env
+rm dev.env
+docker compose up
+```
+</CodeGroup>
 
-3. Run the dbt model block by pressing the play button on the top right of the
-   block or by pressing `Command` + `Enter`.
-4. You should see a preview of the query execution logs. To see the query results,
-   click the `Expand table` link at the bottom right corner.
+After running the above command, you'll see the Mage overview page. Click the pipelines icon on the left to enter the pipelines overview.
 
-![](https://github.com/mage-ai/assets/blob/main/dbt/dbt-preview.gif?raw=true)
+![Open Pipelines](/media/setup-dbt/go-to-pipelines.gif)
 
-5. After previewing the results, in the top right corner of the block,
-   click on the triple dots to reveal a dropdown menu.
-6. Under the dropdown menu, click the option <b>Run model</b>. This command
-   will execute the `dbt run` command and create the table in your data source.
+## Tutorial
 
-![](https://github.com/mage-ai/assets/blob/main/dbt/run-model.png?raw=true)
+<AccordionGroup>
+<Accordion title="A simple dbt pipeline" icon="pipe-collar">
 
-### 6b. Edit dbt model `my_second_dbt_model`
+First, select the `simple_dbt_python_pipeline` by double-clicking— this will take you directly to the editor. You can also single click,
+then select the "code" icon from the side nav.
```

Comment:
Perhaps rephrase to `then select "Edit pipeline" (with the "code" icon) from the side nav`. Just referring to "code icon" might not be super obvious to some.

Reply:
Same here

Reply:
nit: run-on sentence. Period needed after "list of pipelines".

Reply:
> Perhaps rephrase to `then select "Edit pipeline" (with the "code" icon) from the side nav`. Just referring to "code icon" might not be super obvious to some.

this change mightve gotten overwritten